tests/submissions/test_examples.py

```diff
@@ -0,0 +1,169 @@
+import importlib
+import random
+import time
+
+import pytest
+import requests
+
+from hera.exceptions import HeraException
+from hera.workflows.cluster_workflow_template import ClusterWorkflowTemplate
+from hera.workflows.cron_workflow import CronWorkflow
+from hera.workflows.models import Workflow as ModelWorkflow
+from hera.workflows.service import WorkflowsService
+from hera.workflows.workflow import Workflow
+from hera.workflows.workflow_template import WorkflowTemplate
+from tests.test_examples import CI_MODE, _get_examples
+
+# Skip long-running and unsupported examples
+TIME_LIMIT_SECONDS = 120
+SKIP_LINT_EXAMPLES = {
+    "new_decorators_auto_template_refs",  # Uses TemplateRefs to non-existent WorkflowTemplates
+    "parquet_pandas",
+}
+SKIP_SUBMISSION_EXAMPLES = SKIP_LINT_EXAMPLES.union(
```

Comment:
did we want to maybe sort these alphabetically?

Reply:
Might as well 👍